.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - 1.10.x
+  - master
+
+env:
+  - DEP_VERSION="0.4.1"
+
+before_install:
+  # Install dep as per https://golang.github.io/dep/docs/FAQ.html#how-do-i-use-dep-in-ci
+  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
+  - chmod +x $GOPATH/bin/dep
+  # Install metalinter
+  - go get github.com/alecthomas/gometalinter
+  - gometalinter --install
+
+install:
+  - make get-deps
+
+script:
+  - make test
+  - make lint
```

Comment:
Worth adding codecov (if it's just one command)?

Reply:
`make test` already generates the coverage information. The codecov push will accompany the first test -- I don't know what it'll do with an empty coverage report.